contracts/Contribution.sol

```diff
@@ -200,11 +218,25 @@ contract Contribution is Controlled, TokenController {
   // Constant functions
   //////////
 
-  /// @return Total tokens availale for the sale in weis.
+  /// @return Total tokens available for the sale in weis.
   function tokensForSale() public constant returns(uint256) {
     return totalSupplyCap > totalSold ? totalSupplyCap.sub(totalSold) : 0;
   }
 
+  /// @return Total tokens available for a specific address in weis.
+  function tokensForSale(address investor) public constant returns(uint256) {
+    uint256 cap;
+    uint256 sold;
+    if (getBlockTimestamp() <= startTime + 1 days) {
```

Comment:
I don't understand why there is `+ 1 days`

Reply:
good catch, this is meant to be a placeholder for the X hours after (I'm not sure about adding a variable here and all the logic to initiate it)